example/metro.config.windows.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Metro configuration for React Native
+ * https://github.com/facebook/react-native
+ *
+ * @format
+ */
+const fs = require('fs');
+const path = require('path');
+const blacklist = require('metro-config/src/defaults/blacklist');
+
+const rnPath = fs.realpathSync(
+  path.resolve(require.resolve('react-native/package.json'), '..'),
+);
+
+const rnwPath = fs.realpathSync(
+  path.resolve(require.resolve('react-native-windows/package.json'), '..'),
+);
+
+module.exports = {
+  resolver: {
+    extraNodeModules: {
+      // Redirect react-native to react-native-windows
+      'react-native': rnwPath,
+      'react-native-windows': rnwPath,
+    },
+    // Include the macos platform in addition to the defaults because the fork includes macos, but doesn't declare it
+    platforms: ['ios', 'android', 'windesktop', 'windows', 'web', 'macos'],
```

Comment:
what's windesktop? 🤔

Reply:
copied from WebView, which we link to in our docs (https://microsoft.github.io/react-native-windows/docs/native-modules-setup#option-2-adding-windows-support-to-existing-sample-app)

Reply:
Will delete